bjoern.py

```diff
@@ -38,6 +39,18 @@ def server_run(sock, wsgi_app):
     _bjoern.server_run(sock, wsgi_app)
 
 
+def _close_server_socket():
+    # Handler for cleaning up the server socket when the process terminates
+    global _default_instance
+    
+    if sock.family == socket.AF_UNIX:
```

Comment:
Sorry for not reviewing this patch earlier, must have missed it!

Where does the `sock` variable come from? Did you even test this code once? If I'm not terribly mistaken the code doesn't work...

Reply:
Yes I did, as I wound up using it in a production application  and it worked, but it appears that I missed a line when I copied it over from our codebase. My apologies for that.